### DISCUSSION TITLE
Research / Data Contributor

### DISCUSSION THREAD

**Discussion by yliu2019** [BlueQuartzSoftware/simplnx#742] [Contribution Recognition]
Yang Liu, Reseach Associate, Imperial College London. Several side research based on DREAM3D, with 2D and 3D datasets. Potential Data contributor. 
Github: https://github.com/yliu2019

**Comment by nyoungbq** [comment]
We got you put in to the contributors section so you should now be able to see your profile icon in our list at the bottom of the readme. Thank you so much for your contributions and timely responses, feel free to reopen this thread if you have additional contribution categories later on. 

Here's the commit for reference: [#5894c81cc40dd09c00e7d459c05e5f865f2166f7](https://github.com/BlueQuartzSoftware/complex/commit/5894c81cc40dd09c00e7d459c05e5f865f2166f7)
